airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/jsonl_parser.py

```diff
@@ -16,6 +17,7 @@
 class JsonlParser(FileTypeParser):
 
     MAX_BYTES_PER_FILE_FOR_SCHEMA_INFERENCE = 1_000_000
+    _WITH_READ_LIMIT = True
```

Comment:
Do you plan on using this constant more? It feels unnecessary if we're only using it in one place when infer_schema calls `infer_jsonl_entries()`